cypress/integration/report_page_test.js

```diff
@@ -110,4 +114,13 @@ describe("Report Page Test", function(){
         cy.url().should("include","/results");
         cy.get(".im-table").should("exist");
     })
+
+    it("can access other mines", function(){
+        cy.get(".other-mine").within(() => {
+            cy.get(".mine").should("include.text","FlyMine");
+            cy.get("img").should("exist");
+            cy.contains("Eip75B").should("exist").click();
+        })
+        cy.url().should("include","/flymine");
+    })
```

Comment:
Can you also assert that the *D. melanogaster* organism is present and assert the full URL "/flymine/report/Gene/1029368" to ensure the link points to the report page and not just the mine.